tests/net/6lo/prj.conf

```diff
@@ -25,3 +25,4 @@ CONFIG_NET_DEBUG_NET_PKT=y
 CONFIG_NET_6LO_CONTEXT=y
```

Comment:
Typo in commit msg:
s/cleaup/cleanup/

Reply:
Done

Reply:
@jukkar can you please review so that I can merge it in v1.10 ?